dedupeio/dedupe-examples
# Add Gazetteer Postgres example

# Overview

Add a new Gazetteer example to perform static matching backed by a Postgres database with updates.

In addition, containerize the Gazetteer examples.

## Notes

A requirement of this example is to support Dedupe 1.x. Once this PR is approved and merged, and confirmed to work for the folks who requested it, I'll port the example to Dedupe 2.x.

## Testing instructions

* View the [Gazetteer README](https://github.com/dedupeio/dedupe-examples/blob/jfc/gazetteer-pgsql-example/gazetteer_example/README.md)
* Follow the installation instructions and confirm you can build the container images
* Follow the usage instructions and confirm you can run each script
